app/lib/methods/helpers/image.ts

```diff
@@ -0,0 +1,10 @@
+import { Image } from 'react-native';
+
+export const isImageURL = async (url: string) => {
```

Comment:
I'm reviewing from GH only, so Idk if TS is smart enough to infer the return.
In case it doesn't, you should do it yourself, like this.

```suggestion
export const isImageURL = async (url: string): boolean => {
```